sources/Algo/include/GeneratorDefinitionAlgorithm.h

```diff
@@ -134,9 +146,10 @@ class GeneratorDefinitionAlgorithm {
    * @param busesWithDynamicModel map of bus ids to a generator that regulates them
    * @param busMap mapping of busId and the number of generators that regulates them
    * @param infinitereactivelimits parameter to determine if infinite reactive limits are used
+   * @param tfoVoltageLevel maximum voltage level for which we assume that generator's transfos are in the static model
```

Comment:
transformers should be present in the static description

Reply:
we assume that generator's transformers are already described in the static description